### ISSUE TITLE
no-duplicate-constants

### ISSUE THREAD

**Issue by benmosher** [benmosher/eslint-plugin-import#473]
report the following: 

```js
export const x = "x"
export const y = "x"
```

Basically, report all groups of exports that are exporting the same string literal?

This might be too specific for the `import` plugin. I want it for exporting Redux action types, I frequently duplicate a line and forget to update the actual value.

Maybe should be one for `eslint-plugin-redux` (currently just a shell).

**Comment by jfmengels** [comment]
I don't see the value that much, as sometimes you simply need to export duplicate values.
I get the Redux pain for this, but IMO, this is something that your tests should cover. Especially since Redux is something very easy to test.

I think that this would be a problem only in really specific parts of a codebase, and would cause "false" errors in other parts. I'm not even sure this would be a good fit for a redux plugin, unless you can very carefully activate this rule on only those reducer/Redux files (pending a glob based config).

I'd see this more applicable to for instance exporting the same function twice, but then again, sometimes you just need to do that.
